src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -609,32 +607,34 @@ sub-<label>/[ses-<label>/]
         sub-<label>[_ses-<label>][_acq-<label>][_run-<index>]_phasediff.nii[.gz]
         sub-<label>[_ses-<label>][_acq-<label>][_run-<index>]_phasediff.json
         sub-<label>[_ses-<label>][_acq-<label>][_run-<index>]_magnitude1.nii[.gz]
+        sub-<label>[_ses-<label>][_acq-<label>][_run-<index>]_magnitude2.nii[.gz]  # OPTIONAL
```

Comment:
One thing that confuses me about this template (even before this PR)- wouldn't we still have jsons for the magnitude maps?

Reply:
The magnitude maps are part of the acquisition (as the phase maps are), so all the metadata is probably best stored only on the phase information files. I'm not sure what metadata could possibly exist that only applies to the magnitude (and doesn't to the phase).

Reply:
That's true, but I don't think that kind of relationship is supported in the specification. I guess we would call it something like horizontal inheritance?

I could be wrong though.

Reply:
For tools like `pybids`, I think you need sidecar files for the magnitude images in order to extract their metadata. @effigies is that still the case or can `pybids` infer metadata from linked files?

Reply:
I haven't encountered any case where PyBIDS could not correctly query the metadata of fieldmaps.